packages/provider-url-service/src/index.ts

```diff
@@ -19,27 +19,28 @@ interface RequestData {
 }
 
 enum Providers {
-  MOONPAY = 'MOONPAY',
-  RAMP = 'RAMP',
-  TRANSAK = 'TRANSAK',
-  SIMPLEX = 'SIMPLEX',
+  Moonpay = 'Moonpay',
+  Ramp = 'Ramp',
+  Transak = 'Transak',
+  Simplex = 'Simplex',
 }
 
 export const composeCicoProviderUrl = functions.https.onRequest((request, response) => {
   const requestData: RequestData = request.body
   const { provider, address, digitalAsset, fiatCurrency, fiatAmount } = requestData
+  const cashInSuccessDeepLink = `${CASH_IN_SUCCESS_DEEPLINK}/${provider}`
 
   let finalUrl
 
-  if (provider === Providers.MOONPAY) {
+  if (provider === Providers.Moonpay) {
```

Comment:
Since we're changing caps vs non-caps, is it possible this breaks for previous versions? Maybe we could compare calling `.toLowerCase()` on both sides?

Reply:
Yeah it will..this is why I don't like Pascal case in enums. Makes them a lot more annoying to work with because you have to transform both parts of every comparison :(